org.OpenT2T.Onboarding.Hub/opent2tonboarding.js

```diff
@@ -0,0 +1,54 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+
+'use strict';
+
+class accessTokenInfo {
+        constructor(
+            accessToken,
+            refreshToken,
+            tokenType, // ex: 'bearer'
+            scopes // ex: 'full_access'
+        )
+    {
+        this.accessToken = accessToken;
+        this.refreshToken = refreshToken;
+        this.tokenType = tokenType;
+        this.scopes = scopes;
+    }
+}
+
+// Consumers must be aware and handle the case that not all properties are available all the time & for all devices. (check for 'undefined')
+class deviceInfo{

```

Comment:
Can we add a device type/category identifier (e.g. thermostat, light) that can be used to associate the device with translator